src/gentropy/datasource/finngen/finngen_finemapping.py

```diff
@@ -0,0 +1,286 @@
+# pylint: disable=unsubscriptable-object
```

Comment:
I don't think this line is necessary.

Reply:
Without this it throws me an error for type checking where I have list[str]